### DISCUSSION TITLE
How locally link to external package

### DISCUSSION THREAD

**Discussion by fausti-nop** [vercel/turborepo#9143] [Help]
### Summary

Hello,

I'm using two TurboRepos: one manages several libraries, and the other contains various apps with different packages in the same repository. I encounter problems when trying to locally link (npm link @my-package) external packages from the other TurboRepo. If I publish the package and use it from the registry, there are no issues. However, for dynamic development, I need this local linking. Sometimes I manage to make it work, but it breaks when I make changes or reinstall, and trying to relink does not resolve the issue.

Are there any considerations or steps I should be aware of to handle this more effectively?

Thank you in advance!

**Comment by anthonyshew** [comment]
Hi, can you explain Turborepo's role in the issue you're seeing? This appears to be a question that is in the domain of the package manager.
